src/sentry/event_manager.py

```diff
@@ -2512,6 +2517,39 @@ def _detect_performance_problems(
         )
 
 
+@sentry_sdk.tracing.trace
+def _sample_transactions_in_save(jobs: Sequence[Job], projects: ProjectsMapping) -> None:
+    for job in jobs:
+        try:
+            if not in_rollout_group(
+                "transactions.do_post_process_in_save", job["event"].project_id
+            ):
+                continue
+            project = job["event"].project
+            record_transaction_name_for_clustering(project, job["event"].data)
+        except Exception:
+            sentry_sdk.capture_exception()
+
+
+@sentry_sdk.tracing.trace
+def _send_transaction_processed_signals(jobs: Sequence[Job], projects: ProjectsMapping) -> None:
+    for job in jobs:
+        try:
+            if not in_rollout_group(
+                "transactions.do_post_process_in_save", job["event"].project_id
+            ):
+                continue
+
+            project = job["event"].project
+            transaction_processed.send_robust(
+                sender=None,
```

Comment:
i need to determine if setting a sender is important. i don't believe it is though.